%YAML:1.0
stereo0:
  cam0:
    cam_overlaps: [1]
    camera_model: pinhole
    distortion_coeffs: [0.9124923114517718, 4.343265943776822, -0.0009219499901576338, -0.0003758641436705232, 0.5377063691507658, 1.2547773763644394, 4.685659463807727, 1.8914062134393086]
    distortion_model: radtan
    intrinsics: [526.9777898424712, 469.1146463646779, 633.7441796783651, 312.57358183571216]
    resolution: [1280, 640]
    rostopic: /cam0/image_raw
  cam1:
    T_cn_cnm1:
      - [0.99999742810271, -0.001893912930653751, -0.001247750686896911, -0.06982615049924103]
      - [0.0018941825797794797, 0.9999981829302178, 0.00021496189700501392, -0.0001775300800006955]
      - [0.0012473413005305118, -0.0002173248117600904, 0.9999991984544818, -0.0004137320584759926]
      - [0.0, 0.0, 0.0, 1.0]
    cam_overlaps: [0]
    camera_model: pinhole
    distortion_coeffs: [-11.745132167668041, 59.62762717021791, -0.0013560519096700765, -0.0004657014453078306, 12.52780367560598, -11.384858230691012, 55.28872707500754, 34.97449356387675]
    distortion_model: radtan
    intrinsics: [525.3536143155285, 467.8171287466013, 690.010108854434, 304.94870299469596]
    resolution: [1280, 640]
    rostopic: /cam1/image_raw
